src/main/java/subway/line/domain/Line.java

```diff
@@ -31,6 +32,27 @@ public Line(Long id, String name, String color, Sections sections) {
         this.sections = sections;
     }
 
+    public Line(String name, String color, int extraFare) {
+        this.name = name;
+        this.color = color;
+        this.extraFare = extraFare;
+    }
+
+    public Line(Long id, String name, String color, int extraFare) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.extraFare = extraFare;
+    }
```

Comment:
this를 사용하면 해당 파라미터에 적합한 자신의 생성자를 찾아서 쓸수 있어요. 코드량이 조금이라도 줄어듭니다.
```suggestion
public Line(Long id, String name, String color, int extraFare) {
        this(name, color, extraFare);
        this.id = id;
    }